prow/cluster/jobs/istio/pkg/istio.pkg.master.yaml

```diff
@@ -0,0 +1,79 @@
+job_template: &job_template
+  branches:
+    - "^master$"
+  decorate: true
+  path_alias: istio.io/pkg
+
+istio_container: &istio_container
```

Comment:
I don't see any usage of `*istio_container` nor `*job_template`, I'd suggest removing this dead config.

Reply:
I just did  cut & paste from the operator's job definitions. I've removed this, and removed the similar stuff from the operator's config too.